cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -71,7 +80,7 @@ func (c *Controller) spcEventHandler(operation string, spcGot *apis.StoragePoolC
 		break
 
 	case deleteEvent:
-		err := storagepoolactions.DeleteStoragePool(key)
+		err := storagepoolactions.DeleteStoragePool(spcGot)
 
 		if err != nil {
 			glog.Error("Storagepool could not be deleted:", err)
```

Comment:
%v ?

Reply:
No need here... It will append

Reply:
@prateekpandey14

Reply:
Above is `Error` nor `Errorf`

Reply:
That's right ! but its good to have format specifier for error type.